extensions/cornerstone-dicom-sr/src/viewports/OHIFCornerstoneSRViewport.tsx

```diff
@@ -210,9 +210,16 @@ function OHIFCornerstoneSRViewport(props) {
         // should be passed second since we don't want SR displaySet to
         // override the activeImageDisplaySetData
         displaySets={[activeImageDisplaySetData]}
+        // Note: we shouldn't inherit the viewportOptions
```

Comment:
I don't think this commment is completely accurate any longer. The viewport options are inherited save for the toolGroupId and viewport type.

Reply:
We should have a TODO here to revert this change once we switch back to the older mechanism with next/previous measurement buttons, and also add a next/previous measurement keyboard shortcuts

Reply:
Hmm,  we need a viewport type that is volume if possible, otherwise stack.  It should be possible to display the SR as a volume in order to show volume measurements.  For now, forcing to stack is the best choice, but eventually we will need another option.